From f18553790677f5c88364a8d068a4936f21da849f Mon Sep 17 00:00:00 2001
From: "Alam, SahibeX" <sahibex.alam@intel.com>
Date: Thu, 26 Oct 2023 15:09:44 +0530
Subject: [PATCH] Update security_patch_level string

Security_patch_level needs to be updated

When ASB SPL are integrated.
Signed-off-by: Alam, SahibeX <sahibex.alam@intel.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 5acdb497db..1ab04e372e 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -104,7 +104,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2023-10-05
+    PLATFORM_SECURITY_PATCH := 2023-11-01
 endif
 
 include $(BUILD_SYSTEM)/version_util.mk
-- 
2.17.1

